CRM/Contact/Import/Parser/Contact.php

```diff
@@ -1704,4 +1710,42 @@ protected function checkForDuplicateExternalIdentifiers(string $externalIdentifi
     }
   }
 
+  /**
+   * @param string $countyID
+   * @param string|int|null $stateProvinceID
+   * @param string|int|null $countryID
+   *
+   * @return string|int
+   * @throws \CRM_Core_Exception
+   */
+  private function tryToResolveCounty(string $countyID, $stateProvinceID, $countryID) {
+    if (!isset(\Civi::$statics[$countyID . '_' . $stateProvinceID . '_' . $countyID])) {
+      $possibleCounties = $this->ambiguousOptions['county_id'][mb_strtolower($countyID)] ?? NULL;
+      if (!$possibleCounties || $countyID === 'invalid_import_value') {
+        \Civi::$statics[$countyID . '_' . $stateProvinceID . '_' . $countyID] = $countyID;
+      }
+      else {
+        if ($stateProvinceID === NULL && $countryID === NULL) {
+          $countryID = \Civi::settings()->get('defaultContactCountry');
+        }
+        $countyLookUp = County::get(FALSE)
+          ->addWhere('id', 'IN', $possibleCounties);
+        if ($countryID && is_numeric($countryID)) {
+          $countyLookUp->addWhere('state_province_id.country_id', '=', $countryID);
+        }
+        if ($stateProvinceID && is_numeric($stateProvinceID)) {
+          $countyLookUp->addWhere('state_province_id', '=', $stateProvinceID);
+        }
+        $county = $countyLookUp->execute();
+        if (count($county) === 1) {
+          \Civi::$statics[$countyID . '_' . $stateProvinceID . '_' . $countyID] = $county->first()['id'];
```

Comment:
actually all of them

Reply:
@demeritcowboy  - that's why it works - cos they are consistent :-)

But I pushed up a fix

Reply:
Loks Goood.